articles/getting-started-with-browser-sync/index.md

```diff
@@ -0,0 +1,62 @@
+# Getting Started with Browser-sync
+
+
+Browser-sync is an automation tool that makes your web development project faster. It makes your browser testing workflow faster by synchronising URLs, interactions and code changes across multiple devices, both internally and externally.
```

Comment:
developing websites faster would flow better.

Reply:
Thank you, I will work on the adjustment